server/sonar-web/src/main/js/components/common/RestartForm.tsx

```diff
@@ -19,7 +19,7 @@
  */
 import * as React from 'react';
 import * as classNames from 'classnames';
-import Modal from 'react-modal';
+import * as Modal from 'react-modal';
```

Comment:
yeah, looks like the typings are incorrect :(

actually instead of passing `shouldCloseOnEsc={true}` we can just conditionally pass the `onRequestClose` prop!